config/custom-environment-variables.json

```diff
@@ -40,10 +40,12 @@
     "hearings": {
 			"hearingsJurisdictions": "SSCS,PRIVATELAW",
 			"sscs": {
-				"serviceApi": "SERVICES_SSCS_HEARINGS_COMPONENT_API"
+				"serviceApi": "SERVICE_HEARINGS_COMPONENT_API.sscs",
+				"caseTypes": "Appeal"
```

Comment:
The SSCS case type should be "Benefit",
 https://github.com/hmcts/sscs-ccd-definitions/blob/master/benefit/data/sheets/CaseType/CaseType.json

The private law case type(s) need to be confirmed with private law, but appears to be 
PRLAPPS
https://github.com/hmcts/prl-ccd-definitions/blob/master/definitions/private-law/json/CaseType.json

Reply:
Thank you @andywilkinshmcts, I checked the response of the API call /data/internal/cases/[CASE_ID] on the DEMO environment for both SSCS and PrivateLaw and can confirm that the case types you mentioned are correct.

I have updated the code accordingly.